php/Controllers/SettingsController.php

```diff
@@ -168,6 +173,17 @@ class SettingsController {
      */
     private bool $profile_fetch_failed;
 
+    /**
+     * Twig environment
+     */
+    protected $twig;
+
+    /**
+     * Twig templates directory path
+     * @var string twig_templates_dir_path
+     */
+    const twig_templates_dir_path = COYOTE_PLUGIN_PATH . 'php' . DIRECTORY_SEPARATOR . 'Views';
```

Comment:
This looks like a `PluginConfiguration` candidate.

Reply:
Agreed, I've redefined it there as a constant or would a get function be more desirable?

Reply:
A constant is fine, this is an otherwise immutable value.